src/doc/gui_manual/Quantitative/Expressions.rst

```diff
@@ -942,7 +947,7 @@ Array Expressions
 
 .. _Array_Compose_Expression_Function:
 
-Array Compose Function: ``array_compose()`` : ``array_compose(expr0, expr1, ..., exprN-1)``
+Array Compose Function: ``array_compose()`` : ``array_compose(expr1, exprN, ..., exprK)``
```

Comment:
I don't understand this change. What does "expr1, exprN, ..., exprK" mean?

Reply:
Whoops. that should be `expr2`, not `exprN`. I switched from `N` to `K` for the number of items too.